### PR TITLE
Node geolocation

### PR DESCRIPTION
**Pull request by MiguelSuay**
Adding node-geolocation code to master branch

### CODE REVIEW

File: codigo/node-geolocation/app.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+let googleApiKey = process.env.GOOGLE_API_KEY
+
+let start = argv.start
+let end = argv.end
+
+//Valencian turistic apartments, to geolocate with google.
+csv()
+.fromFile(csvFilePath)
+.then(async (jsonObj)=>{
+    console.log(jsonObj[0]);
```
Comment (datos-solidarios) [inline codigo/node-geolocation/app.js]: Remove console.log()

File: codigo/node-geolocation/geolocate.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const request = require('request-promise')
+const utf8 = require('utf8');
+require('dotenv').config()
+let apiKey = process.env.GEOCODING_API_KEY
+
+
+let geolocate = async (address) => {
+  let curatedGeolocation = {}
```
Comment (datos-solidarios) [inline codigo/node-geolocation/geolocate.js]: Better naming. curatedGeolocationResponse?

File: codigo/node-geolocation/geolocate.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const request = require('request-promise')
+const utf8 = require('utf8');
+require('dotenv').config()
```
Comment (datos-solidarios) [inline codigo/node-geolocation/geolocate.js]: To the top

File: codigo/node-geolocation/geolocate.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    return body
+  })
+  geolocationData = get(["Response","View",0,"Result",0,"Location"], JSON.parse(geolocationData))
+  if (geolocationData){
+    curatedGeolocation.displayCoordenates = geolocationData.DisplayPosition
+    curatedGeolocation.navCoordenates = geolocationData.NavigationPosition
+    curatedGeolocation.apiAddress = geolocationData.Address.Label
+    curatedGeolocation.postalCode = geolocationData.Address.PostalCode
+    curatedGeolocation.apiLocationId = geolocationData.LocationId
+    console.log(JSON.stringify(curatedGeolocation))
```
Comment (datos-solidarios) [inline codigo/node-geolocation/geolocate.js]: Remove

File: codigo/node-geolocation/geolocate.js (hunk 2 above)
Comment (MiguelSuay) [reply codigo/node-geolocation/geolocate.js]: Too long. I think is specific enough.

**Pull request merged**